src/toolbar/Toolbar.jsx

```diff
@@ -91,17 +107,23 @@ export const Toolbar = props => (
         .filter(toolbarGroupHasContent)
         .map((group, index) =>
           /* eslint react/no-array-index-key: "off" */
-          <ToolbarGroup key={index} onClick={props.onClick} group={collapseCustomGroups(group)} />)
+          <ToolbarGroup key={index} onClick={props.onClick} group={collapseCustomGroups(group, props.kebabLimit)} />)
```

Comment:
I suppose we don't have anything better than index right?

Reply:
Not for the group. But the index is stable.